server/service/osquery_utils/queries.go

```diff
@@ -1415,6 +1437,47 @@ func directIngestDiskEncryptionKeyDarwin(
 	return ds.SetOrUpdateHostDiskEncryptionKey(ctx, host.ID, rows[0]["filevault_key"])
 }
 
+func directIngestDiskEncryptionKeyFileLinesDarwin(
+	ctx context.Context,
+	logger log.Logger,
+	host *fleet.Host,
+	ds fleet.Datastore,
+	rows []map[string]string,
+) error {
+	if len(rows) == 0 {
+		// assume the extension is not there
+		level.Debug(logger).Log(
+			"component", "service",
+			"method", "directIngestDiskEncryptionKeyFileLinesDarwin",
+			"msg", "no rows or failed",
+			"host", host.Hostname,
+		)
+		return nil
+	}
+
+	var hexLines []string
+	for _, row := range rows {
```

Comment:
nit: should we add a comment here for our future selves explaining why we're doing the concatenation in Go? a quick summary of what we found out